<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-3IDCSOL" failures="1" tests="3" name="Default test(failed)" time="0.111" errors="0" timestamp="2022-06-15T06:51:22 GMT-08:00">
  <testcase classname="test_NG_XMLstudy.failmethodclass" name="Method7" time="0.014"/>
  <testcase classname="test_NG_XMLstudy.failmethodclass" name="Method6" time="0.004"/>
  <testcase classname="test_NG_XMLstudy.failmethodclass" name="Method10" time="0.006">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test_NG_XMLstudy.failmethodclass.Method10(failmethodclass.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Method10 -->
  <testcase classname="test_NG_XMLstudy.failmethodclass" name="Method9" time="0.0">
    <ignored/>
  </testcase> <!-- Method9 -->
  <testcase classname="test_NG_XMLstudy.failmethodclass" name="Method8" time="0.0">
    <ignored/>
  </testcase> <!-- Method8 -->
</testsuite> <!-- Default test(failed) -->
